devs/docs/es-backports.rst

```diff
@@ -487,28 +487,28 @@ should be crossed out as well.
 - [ ] b6c94fd73e8 Fix Term Vectors with artificial docs and keyword fields (#53504) (#53550)
 - [ ] 2438b899eb0 Support joda style date patterns in 7.x (#52555)
 - [ ] 9ada5083479 Fix date_nanos in composite aggs (backport of #53315) (#53347)
-- [ ] ac721938c22 Allow joining node to trigger term bump (#53338)
+- [x] ac721938c22 Allow joining node to trigger term bump (#53338)
 - [ ] 7189c57b6cb Record Force Merges in Live Commit Data (#52694) (#53372)
 - [x] 24f114766fb Fix doc_stats and segment_stats of ReadOnlyEngine (#53345)
-- [ ] 5c861cfe6e2 Upgrade to final lucene 8.5.0 snapshot (#53293)
+- [s] 5c861cfe6e2 Upgrade to final lucene 8.5.0 snapshot (#53293)
 - [x] 5e96d3e59ae Use given executor for global checkpoint listener (#53260)
-- [ ] c5738ae312a Notify refresh listeners on the calling thread (#53259)
-- [ ] f68917160e8 Fix RemoteConnectionManager size() method (#52823)
-- [ ] c610e0893db Introduce system index APIs for Kibana (#53035)
+- [s] c5738ae312a Notify refresh listeners on the calling thread (#53259)
+- [s] f68917160e8 Fix RemoteConnectionManager size() method (#52823)
+- [s] c610e0893db Introduce system index APIs for Kibana (#53035)
 - [x] a154f9c657d Early return if no global checkpoint listeners (#53036)
-- [ ] 331d4bb0afe HybridDirectory should mmap postings. (#52641) (#52873)
-- [ ] 6aa9aaa2c62 Add validation for dynamic templates (#52890)
-- [ ] 1d1956ee93d Add size support to `top_metrics` (backport of #52662) (#52914)
-- [ ] 52fa4653003 Cache completion stats between refreshes (#52872)
+- [s] 331d4bb0afe HybridDirectory should mmap postings. (#52641) (#52873)
```

Comment:
maybe also as `[x]`, I think this is applied already.

Reply:
It's kinda fuzzy in this case because we didn't apply the patch specificly but did our version. But it also doesn't matter too much. We're going to prune/remove this eventually anyways.  Going with either is fine